web/src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -241,6 +249,23 @@ export default {
     ...mapGetters('dandiset', ['version']),
   },
   asyncComputed: {
+    userCanModifyDandiset: {
+      async get() {
+        if (this.user.admin) {
+          return true;
+        }
```

Comment:
I wonder if we should disallow admins from editing metadata at this point. Do we need to have this backdoor access?

Reply:
I think it's useful for admins to be able to step in and tweak metadata if people have issues or questions.

Reply:
You're right. Maybe we can add something in the UI that shows you only have access because you're an admin. That's not important right now and if I care enough about it I'll file an issue.